docs/README.rst

```diff
@@ -4,56 +4,9 @@
 Scrapy documentation quick start guide
 ======================================
 
-This file provides a quick guide on how to compile the Scrapy documentation.
-
-
-Setup the environment
----------------------
-
-To compile the documentation you need Sphinx Python library. To install it
-and all its dependencies run the following command from this dir
-
-::
-
-    pip install -r requirements.txt
-
-
-Compile the documentation
--------------------------
-
 To compile the documentation (to classic HTML output) run the following command
 from this dir::
 
-    make html
```

Comment:
I don't know about this, it is in fact how I'm building the docs these days. `tox -e docs` gives me the following error, and I have stopped trying to understand how Sphinx works:

```
$ tox -e docs
GLOB sdist-make: /.../scrapy/setup.py
docs inst-nodeps: /.../scrapy/.tox/.tmp/package/1/Scrapy-2.1.0.zip
docs installed: alabaster==0.7.12,attrs==19.3.0,Automat==20.2.0,Babel==2.8.0,certifi==2020.4.5.1,cffi==1.14.0,chardet==3.0.4,constantly==15.1.0,cryptography==2.9.2,cssselect==1.1.0,docutils==0.16,hyperlink==19.0.0,idna==2.9,imagesize==1.2.0,incremental==17.5.0,Jinja2==2.11.2,lxml==4.5.0,MarkupSafe==1.1.1,packaging==20.3,parsel==1.6.0,Protego==0.1.16,pyasn1==0.4.8,pyasn1-modules==0.2.8,pycparser==2.20,PyDispatcher==2.0.5,Pygments==2.6.1,PyHamcrest==2.0.2,pyOpenSSL==19.1.0,pyparsing==2.4.7,pytz==2020.1,queuelib==1.5.0,requests==2.23.0,Scrapy==2.1.0,service-identity==18.1.0,six==1.14.0,snowballstemmer==2.0.0,Sphinx==3.0.3,sphinx-hoverxref==0.3b1,sphinx-notfound-page==0.4,sphinx-rtd-theme==0.4.3,sphinxcontrib-applehelp==1.0.2,sphinxcontrib-devhelp==1.0.2,sphinxcontrib-htmlhelp==1.0.3,sphinxcontrib-jsmath==1.0.1,sphinxcontrib-qthelp==1.0.3,sphinxcontrib-serializinghtml==1.1.4,Twisted==20.3.0,urllib3==1.25.9,w3lib==1.21.0,zope.interface==5.1.0
docs run-test-pre: PYTHONHASHSEED='2542448692'
docs run-test: commands[0] | sphinx-build -W -b html . /.../scrapy/.tox/docs/tmp/html
Running Sphinx v3.0.3
loading translations [en]... done
/.../scrapy/.tox/docs/lib/python3.6/site-packages/sphinx/deprecation.py:52: RemovedInSphinx40Warning: sphinx.builders.html.DirectoryHTMLBuilder is deprecated. Check CHANGES for Sphinx API modifications.
  return getattr(self._module, name)
making output directory... done
loading intersphinx inventory from https://coverage.readthedocs.io/en/stable/objects.inv...
loading intersphinx inventory from https://cssselect.readthedocs.io/en/latest/objects.inv...
loading intersphinx inventory from https://docs.pytest.org/en/latest/objects.inv...
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://www.sphinx-doc.org/en/master/objects.inv...
loading intersphinx inventory from https://tox.readthedocs.io/en/latest/objects.inv...
loading intersphinx inventory from https://twistedmatrix.com/documents/current/objects.inv...
loading intersphinx inventory from https://twistedmatrix.com/documents/current/api/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 80 source files that are out of date
updating environment: [new config] 80 added, 0 changed, 0 removed
reading sources... [  1%] README
reading sources... [  2%] contributing
reading sources... [  3%] faq
reading sources... [  5%] index
reading sources... [  6%] intro/examples
reading sources... [  7%] intro/install
reading sources... [  8%] intro/overview
reading sources... [ 10%] intro/tutorial
reading sources... [ 11%] news
reading sources... [ 12%] topics/api
reading sources... [ 13%] topics/architecture
reading sources... [ 15%] topics/asyncio
reading sources... [ 16%] topics/autothrottle
reading sources... [ 17%] topics/benchmarking
reading sources... [ 18%] topics/broad-crawls
reading sources... [ 20%] topics/commands
reading sources... [ 21%] topics/contracts
reading sources... [ 22%] topics/coroutines
reading sources... [ 23%] topics/debug
reading sources... [ 25%] topics/deploy
reading sources... [ 26%] topics/developer-tools
reading sources... [ 27%] topics/djangoitem
reading sources... [ 28%] topics/downloader-middleware
reading sources... [ 30%] topics/dynamic-content
reading sources... [ 31%] topics/email
reading sources... [ 32%] topics/exceptions
reading sources... [ 33%] topics/exporters
reading sources... [ 35%] topics/extensions
reading sources... [ 36%] topics/feed-exports
reading sources... [ 37%] topics/item-pipeline
reading sources... [ 38%] topics/items
reading sources... [ 40%] topics/jobs
reading sources... [ 41%] topics/leaks
reading sources... [ 42%] topics/link-extractors
reading sources... [ 43%] topics/loaders
reading sources... [ 45%] topics/logging
reading sources... [ 46%] topics/media-pipeline
reading sources... [ 47%] topics/practices
reading sources... [ 48%] topics/request-response
reading sources... [ 50%] topics/scrapyd
reading sources... [ 51%] topics/selectors
reading sources... [ 52%] topics/settings
reading sources... [ 53%] topics/shell
reading sources... [ 55%] topics/signals
reading sources... [ 56%] topics/spider-middleware
reading sources... [ 57%] topics/spiders
reading sources... [ 58%] topics/stats
reading sources... [ 60%] topics/telnetconsole
reading sources... [ 61%] topics/webservice
reading sources... [ 62%] venv-docs/lib/python3.7/site-packages/Jinja2-2.11.2.dist-info/LICENSE
reading sources... [ 63%] venv-docs/lib/python3.7/site-packages/alabaster-0.7.12.dist-info/DESCRIPTION
reading sources... [ 65%] venv-docs/lib/python3.7/site-packages/chardet-3.0.4.dist-info/DESCRIPTION
reading sources... [ 66%] venv-docs/lib/python3.7/site-packages/constantly-15.1.0.dist-info/DESCRIPTION
reading sources... [ 67%] venv-docs/lib/python3.7/site-packages/cryptography-2.9.2.dist-info/AUTHORS
reading sources... [ 68%] venv-docs/lib/python3.7/site-packages/hyperlink-19.0.0.dist-info/DESCRIPTION
reading sources... [ 70%] venv-docs/lib/python3.7/site-packages/idna-2.9.dist-info/LICENSE
reading sources... [ 71%] venv-docs/lib/python3.7/site-packages/imagesize-1.2.0.dist-info/LICENSE
reading sources... [ 72%] venv-docs/lib/python3.7/site-packages/incremental-17.5.0.dist-info/DESCRIPTION
reading sources... [ 73%] venv-docs/lib/python3.7/site-packages/pyasn1-0.4.8.dist-info/LICENSE
reading sources... [ 75%] venv-docs/lib/python3.7/site-packages/pytz-2020.1.dist-info/DESCRIPTION
reading sources... [ 76%] venv-docs/lib/python3.7/site-packages/queuelib-1.5.0.dist-info/DESCRIPTION
reading sources... [ 77%] venv-docs/lib/python3.7/site-packages/sphinx/ext/autosummary/templates/autosummary/base
reading sources... [ 78%] venv-docs/lib/python3.7/site-packages/sphinx/ext/autosummary/templates/autosummary/class
reading sources... [ 80%] venv-docs/lib/python3.7/site-packages/sphinx/ext/autosummary/templates/autosummary/module
reading sources... [ 81%] venv-docs/lib64/python3.7/site-packages/Jinja2-2.11.2.dist-info/LICENSE
reading sources... [ 82%] venv-docs/lib64/python3.7/site-packages/alabaster-0.7.12.dist-info/DESCRIPTION
reading sources... [ 83%] venv-docs/lib64/python3.7/site-packages/chardet-3.0.4.dist-info/DESCRIPTION
reading sources... [ 85%] venv-docs/lib64/python3.7/site-packages/constantly-15.1.0.dist-info/DESCRIPTION
reading sources... [ 86%] venv-docs/lib64/python3.7/site-packages/cryptography-2.9.2.dist-info/AUTHORS
reading sources... [ 87%] venv-docs/lib64/python3.7/site-packages/hyperlink-19.0.0.dist-info/DESCRIPTION
reading sources... [ 88%] venv-docs/lib64/python3.7/site-packages/idna-2.9.dist-info/LICENSE
reading sources... [ 90%] venv-docs/lib64/python3.7/site-packages/imagesize-1.2.0.dist-info/LICENSE
reading sources... [ 91%] venv-docs/lib64/python3.7/site-packages/incremental-17.5.0.dist-info/DESCRIPTION
reading sources... [ 92%] venv-docs/lib64/python3.7/site-packages/pyasn1-0.4.8.dist-info/LICENSE
reading sources... [ 93%] venv-docs/lib64/python3.7/site-packages/pytz-2020.1.dist-info/DESCRIPTION
reading sources... [ 95%] venv-docs/lib64/python3.7/site-packages/queuelib-1.5.0.dist-info/DESCRIPTION
reading sources... [ 96%] venv-docs/lib64/python3.7/site-packages/sphinx/ext/autosummary/templates/autosummary/base
reading sources... [ 97%] venv-docs/lib64/python3.7/site-packages/sphinx/ext/autosummary/templates/autosummary/class
reading sources... [ 98%] venv-docs/lib64/python3.7/site-packages/sphinx/ext/autosummary/templates/autosummary/module
reading sources... [100%] versioning


Warning, treated as error:
/.../scrapy/docs/venv-docs/lib/python3.7/site-packages/sphinx/ext/autosummary/templates/autosummary/base.rst:3:Error in "currentmodule" directive:
maximum 1 argument(s) allowed, 3 supplied.

.. currentmodule:: {{ module }}
ERROR: InvocationError for command /.../scrapy/.tox/docs/bin/sphinx-build -W -b html . /.../scrapy/.tox/docs/tmp/html (exited with code 2)
__________________________________________________________________________________________________________________ summary ___________________________________________________________________________________________________________________
ERROR:   docs: commands failed
```